### PR TITLE
Add multiple updates to increase user experience

### PR DESCRIPTION
**Pull request by coder-extreme**
As requested by @cTheDragons, I have made the updates described below:

- I added the ability to delete messages that you created

- I allowed you to like other people's posts

- I allowed you to flag inappropriate posts (I don't think there is a way to tell the reason using the Habitica API)

- I fixed a couple of display issues (Such as the chat placeholder not showing up)

- I added Contributor Tier label colors and icons

- I made several changes to help with the transition to Usernames including usernames being displayed beside the Display Name and using usernames for referrals instead of display names.

### CODE REVIEW

**Comment by coder-extreme** [comment]
Sorry! I realized I was using two different accounts, one for commits and one for pull requests. That's odd...

**Comment by cTheDragons** [comment]
@crookedneighbor I'm not sure how long it takes to release the this. That being said, it may be an idea to release it now as we only have a limited time with  @coder-extreme. (Another 10 days), 

Not sure you been following the chat in guild,  this version is relatively bug free. The only bug outstanding is the animal ears do not show on the Avatar. 

There are plans to bring in 3 more features, however if there are any bugs it would be good to know of any other issues now, before @coder-extreme finishes. 

Thanks

**Comment by coder-extreme** [comment]
I have finished all of the edits that have been requested.

I have also ported to firefox [here](https://github.com/coder-extreme/habitica-chat-extension-firefox).

**Comment by coder-extreme** [comment]
Sorry, one more bug to fix.

File: manifest.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "name": "Habitica Chat Client",
   "short_name": "Habit Chat",
   "description": "This extension adds a Habitica chat client to habitica.com",
-  "version": "1.4.8",
+  "version": "1.8.6",
```
Comment (crookedneighbor) [inline manifest.json]: Why the large jump here? Shouldn't this be 1.5.0?

**Comment by cTheDragons** [comment]
@crookedneighbor There has been some testing that occurring in the background and to assist with us tracking bugs we need the version number to increase.

I just noticed you missed out on the invite to the testing guild. Apologies I have now invited you. 
There is a discussion regarding ongoing Firefox and Chrome repos that you may be interested in.

Did you want a full list of changes and the issues it fixes with this pull request?

**Comment by cTheDragons** [comment]
This is the last update to this pull request.

Next pull request will include the Firefox port, update to all files etc.

**Comment by coder-extreme** [comment]
This is dealt with in #37, and I was asked to close this as a result.

**Pull request closed without merging**